helsingborg-stad/helsingborg-io-sls-api
# fix: response format from the version-api now also includes updateUrl

## Explain the changes you’ve made
Changed the return format from getStatus lambda to now also includes updateUrl.

Also moved constants.js file to helpers folder to align with other services.

## Explain why these changes are made
We do not want the application itself to hold these kind of values in case they would change. By having them specified in backend we could easily change and update them to all environments.

## Explain your solution
the response is now an object containing `status` and `updateUrl` properties instead of only string.

## How to test
1. Checkout this branch
2. run `sls deploy` within service folder for `service x`
3. Make sure that your SSM has versionEnvs configure
4. Run the lambda in postman or application

## Tested environments

- [] Your local Serverless environment.
